### ISSUE TITLE
Issue with Velux skylight since firmware 2024.3.3-10 - attributes not updated after auto close due to rain

### ISSUE THREAD

**Issue by BernoldNL** [Somfy-Developer/Somfy-TaHoma-Developer-Mode#138]
As I was told in issue #109 that my new issue is not related, I'm creating a new issie.

I noticed that after updating the TaHoma Switch to the latest firmware (2024.3.3-10) there are some changes to io:WindowOpenerVeluxIOComponent:

When closed automatically (in this case due to rain), the core:ClosureState is not updated anymore. It should go to 100%, but it stays at the value it was. Also core:OpenClosedState stays at "open", so there is no way of knowing that it actually closed.

 The Velux product is a skylight (CVU, originally powered by powerline, but after a callback last year it's solar powered), so not a window, which closes 100% when it rains.

**Comment by llavorel-somfy** [comment]
Hi @BernoldNL 

Are you still facing this issue ?

My understanding is that this attribute has never been proactively updated... but I might be mistaking.

**Comment by BernoldNL** [comment]
I would say I'm probably still facing this issue. But this skylight is usually only opened in the summer to let the heat out, so it hasn't been open for months. Also, it's been dry here for a long time.

Give me a few days to update my TaHoma Switch to the latest version, open the skylight and crawl on the roof to sprinkle the sensor with water.

I am pretty sure that the core:ClosureState and core:OpenClosedState attributes were updated when the skylight closes automatically before firmware 2024.3.3-10. And my partner was the first to notice ("Why is the skylight closed, but your app says it's open?"). She's never wrong about these things :)

**Comment by BernoldNL** [comment]
I checked. The issue is still the same:

- I opened the skylight fully. States core:ClosureState went to 0 and core:OpenClosedState to open.
- Made the sensor wet. The skylight closed automatically. States core:ClosureState still showed 0 and core:OpenClosedState still showed open.
- Used the close command and within 2 seconds core:ClosureState showed 100 and core:OpenClosedState showed closed.

**Comment by BernoldNL** [comment]
Has this by any chance been fixed in the last firmware, or maybe the one before that (We haven't had much rain lately..)?

My skylight seems to have closed automatically this morning due to rain and it looks like core:ClosureState and core:OpenClosedState did get updated. But I'm not 100% sure and I won't be able to check for the next few days.

**Comment by qelanhari** [comment]
Throw water at the sensor 😅

**Comment by BernoldNL** [comment]
> Throw water at the sensor 😅

Water is being thrown at the sensor all morning.. 🙄
Don't think it will open.
Also, I can't throw 100km or so 😁

**Comment by BernoldNL** [comment]
Just came across this issue again and saw I had not tested anymore since July. I just tried again. The results are the same as on April 14 (https://github.com/Somfy-Developer/Somfy-TaHoma-Developer-Mode/issues/138#issuecomment-2801350784)
So, please disregard my comment from July 16.

TL;DR Still facing this issue with 2025.5.5-7

**Comment by llavorel-somfy** [comment]
@BernoldNL I fear your version of Velux window is not proactively pushing events in the case of status change due to standalone automatism (water security).

You interpret this as a regression but I don't see why ...
